src/api/public/apidocs-new/paths/source_project_name_package_name_meta.yaml

```diff
@@ -0,0 +1,175 @@
+get:
+  summary: Get package meta file.
+  description: Get package meta file.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: meta
+      schema:
+        type: string
+      description: Set to `1` to force to retrieve the package metadata from the backend.
+      example: 1
+    - in: query
+      name: rev
+      schema:
+        type: string
+      description: Return the package metadata for a specific revision number.
+      example: 
```

Comment:
The `rev` example is empty here, therefore Swagger UI is displaying `Example : null`.

Reply:
Ops, true. Fixed.